packages/documentation/package.json

```diff
@@ -33,23 +33,22 @@
     "gatsby-transformer-sharp": "^2.9.0",
     "gh-pages": "^3.0.0",
     "node-github-graphql": "^0.2.7",
-    "node-sass": "^4.11.0",
     "prop-types": "^15.6.2",
     "puppeteer": "^2.0.0",
     "react": "^16.6.3",
     "react-dom": "^16.6.3",
     "react-helmet": "^6.1.0",
     "react-live": "1.12.0",
     "uswds": "1.6.10",
-    "web-components": "https://github.com/department-of-veterans-affairs/component-library.git#wc-v0.5.0"
+    "web-components": "https://github.com/department-of-veterans-affairs/component-library.git#wc-v0.9.0"
```

Comment:
Not strictly speaking necessary for what I was trying to do, but while I was here. :man_shrugging: